src/modules/launcher/Plugins/Microsoft.Plugin.Folder/Main.cs

```diff
@@ -106,6 +109,22 @@ public static List<Result> GetFolderPluginResults(Query query)
             return results;
         }
 
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Design", "CA1031:Do not catch general exception types", Justification = "We want to keep the process alive and instead inform the user of the error")]
+        private static bool OpenFileOrFolder(string program, string path)
+        {
+            try
+            {
+                Process.Start(program, path);
+            }
+            catch (Exception ex)
+            {
+                string messageBoxTitle = string.Format(CultureInfo.InvariantCulture, "{0} {1}", Properties.Resources.wox_plugin_folder_select_folder_OpenFileOrFolder_error_message, path);
+                MessageBox.Show(ex.Message, messageBoxTitle);
+            }
+
```

Comment:
This is the general format in which exceptions are logged in our codebase. Maybe we'd want to stick with the same.

```
catch (Exception e)
{
    var message = $"Fail to open file at {record.FullPath}";
    LogException(message, e);
    _context.API.ShowMsg(message);\
}
```